qtpy/tests/test_qtcore.py

```diff
@@ -16,34 +16,41 @@
 )
 from qtpy.tests.utils import not_using_conda
 
+NOW = datetime.now()
+# Make integer milliseconds; `floor` here, don't `round`!
+NOW = NOW.replace(microsecond=(NOW.microsecond // 1000 * 1000))
```

Comment:
To avoid double-assignment of a module-level constant, why not

```suggestion
# Make integer milliseconds; `floor` here, don't `round`!
NOW = datetime.now().replace(microsecond=(NOW.microsecond // 1000 * 1000))
```

Reply:
There is the second `NOW` there.

Reply:
Oh, d'oh! In that case, you _could_ do...


```suggestion
_now = datetime.now()
# Make integer milliseconds; `floor` here, don't `round`!
NOW = _now.replace(microsecond=(_now.microsecond // 1000 * 1000))
```

Reply:
Done: https://github.com/spyder-ide/qtpy/pull/421/commits/6ee0931475bc2c33c93d9365ed1d51b7a65f11a1